src/Microsoft.Health.Fhir.SqlServer/Features/Search/SqlServerSearchService.cs

```diff
@@ -239,7 +241,8 @@ private async Task<SearchResult> SearchImpl(SearchOptions searchOptions, bool hi
                                 null,
                                 null,
                                 null),
-                            isMatch ? SearchEntryMode.Match : SearchEntryMode.Include));
+                            isMatch ? SearchEntryMode.Match : SearchEntryMode.Include,
+                            isPartial));
```

Comment:
I think that isPartial belongs more on the `SearchResult` than the individual entries.